README.md

```diff
@@ -174,7 +174,7 @@ context.router.popUntilRoot();
 // removes the entry.
 context.router.removeLast(); 
 
-// removes any route in stack that satisfis the predicate
+// removes any route in stack that satisfies the predicate
 // this works exactly like removing items from a regualar List
```

Comment:
regualar -> regular too 👍

Reply:
I pushed a new commit with some additional grammar & typo fixes, including this one as well 😁